Compression/src/CompressDigiMCs_module.cc

```diff
@@ -541,6 +555,30 @@ void mu2e::CompressDigiMCs::produce(art::Event & event)
     }
   }
 
+  // Now compress the SurfaceSteps
+  for (const auto& surfaceStepsTag : _surfaceStepTags) {
```

Comment:
the plan is to produce 1 SurfaceStep collection. If having a sequence of tags makes things more consistent this is fine, otherwise a single value would also be fine.

Reply:
Thanks, Dave. A sequence of tags is more consistent with what StepPointMCs has. It also gives us some flexibility so that we can combine SurfaceSteps from multiple inputs like backround frames (although maybe that should be done in the MixBackgroundFrames module...)